assets/vsphere_problem_detector/12_prometheusrules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: vsphere-problem-detector
+  namespace: openshift-cluster-storage-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: vsphere-problem-detector.rules
+      rules:
+      - alert: VSphereOpenshiftNodeHealthFail
+        expr:  vsphere_node_check_errors == 1
+        for: 10m
+        labels:
+          severity: warning
+        annotations:
+          message: "VSphere node health checks are failing on {{ $labels.node }} with {{ $labels.check }}"
+      - alert: VSphereOpenshiftClusterHealthFail
+        expr: vsphere_cluster_check_errors == 1
+        for: 10m
+        labels:
+          severity: critical
```

Comment:
The other one is `warning`, do we really want it critical? I'd start as low as possible, we don't know how many clusters are going to report the alert after upgrade to 4.7

Reply:
changed this to warning as well.